SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -472,3 +485,180 @@ void USpatialWorkerConnection::OnConnectionFailure()
 		GEngine->BroadcastNetworkFailure(GameInstance->GetWorld(), GetSpatialNetDriverChecked(), ENetworkFailure::FromDisconnectOpStatusCode(ConnectionStatusCode), *ErrorMessage);
 	}
 }
+
+bool USpatialWorkerConnection::Init()
+{
+	return true;
+}
+
+uint32 USpatialWorkerConnection::Run()
+{
+	while (KeepRunning)
+	{
+		FPlatformProcess::Sleep(0.015f);
```

Comment:
This is not sleeping long enough imo.

Reply:
Totally. Just was there for testing. Was going to also bring up, should this be configurable?

Reply:
No doesn't need to be configurable at this stage unless there is explicit need from a customer. Make it 30hz?

Reply:
Discussed offline, let's make this configurable.